reference/latest/src/main/java/com/example/docs/event/Events.java

```diff
@@ -0,0 +1,58 @@
+package com.example.docs.event;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.block.Blocks;
+import net.minecraft.entity.ItemEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.loot.LootPool;
+import net.minecraft.loot.entry.ItemEntry;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.api.event.player.AttackBlockCallback;
+import net.fabricmc.fabric.api.loot.v2.LootTableEvents;
+
+// Class to contain all mod events.
+public class Events {
+	private static final Identifier COAL_ORE_LOOT_TABLE_ID = Blocks.COAL_ORE.getLootTableId();
+
+	// Call this method in your initializer.
+	public static void registerModEvents() {
```

Comment:
Would it be better to make this class a ModInitializer?

Reply:
Probably, yeah. Might be worth compartmentalizing pages into seperate initializers for future reference.